test/integration/conftest.py

```diff
@@ -50,17 +51,33 @@ def realm() -> str:
 
 @pytest.fixture(scope='function')
 def bucket_name(b2_api) -> str:
-    yield b2_api.create_bucket().name
-
-
-@pytest.fixture(scope='session')
-def general_bucket_name_prefix() -> str:
-    yield 'clitst'
+    bucket = b2_api.create_bucket()
+    yield bucket.name
+    try:
+        b2_api.clean_bucket(bucket)
+    except BucketIdNotFound:
+        pass
```

Comment:
Legacy. We had some issues with concurrency and it was supposed to fix it. Redundant now.

Reply:
Hm, actually there was another case: if for some reason test wants to delete bucket itself

Reply:
This may happen if two cleanup runs are running at the same time. Usually shouldn't happen, but doesn't hurt a whole lot to keep it here.
maybe rewrite to 
```
with contextlib.suppress(BucketIdNotFound):
    b2_api.clean_bucket(bucket)
```